### DISCUSSION TITLE
Button inside popover is not visible

### DISCUSSION THREAD

**Discussion by muhammadsaeed21** [twbs/bootstrap#34640] [Q&A] [answered]
I am using heading and button in data content but only heading is visible and the button is not visible.

<button id="btn-pop" data-html="true" class="btn btn-secondary" data-container="body" data-toggle="popover" data-placement="bottom" data-content="<h1>Heading</h1><button class='btn btn-primary'>Btn</button>">
  Popover on bottom
</button>

<script>
$('#btn-pop').popover();
</script>

**Comment by alpadev** [comment, marked as answer]
I think you need to use `data-bs-html="true"` for this to work. And also add the `button` elements to the `allowList` or the [sanitizer](https://getbootstrap.com/docs/5.0/getting-started/javascript/#sanitizer) will remove it (alternatively use an `<a/>` element or disable the sanitization).

Here is some example: https://jsfiddle.net/y87sam34/

**Reply by muhammadsaeed21** [reply]
Thanks

**Reply by windchimee** [reply]
I think good way to start. Will Vic's work for this issue as well? Respiratory infection.
